hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleAccumulator.java

```diff
@@ -543,7 +555,8 @@ private ThrottleGroup<HederaFunctionality> hapiGroupFromPbj(
     }
 
     private void logResolvedDefinitions(final int capacitySplit) {
-        var sb = new StringBuilder("Resolved handle throttles (after splitting capacity ")
+        var sb = new StringBuilder("Resolved " + throttleType + " ")
```

Comment:
Concatenating a String and passing the result to a StringBuilder is inefficient. It is better to use `append()` already here.